databuilder/extractor/hive_table_metadata_extractor.py

```diff
@@ -27,7 +27,7 @@ class HiveTableMetadataExtractor(Extractor):
     # Using UNION to combine above two statements and order by table & partition identifier.
     SQL_STATEMENT = """
     SELECT source.* FROM
-    (SELECT t.TBL_ID, d.NAME as schema, t.TBL_NAME name, t.TBL_TYPE, tp.PARAM_VALUE as description,
+    (SELECT t.TBL_ID, d.NAME as schema_name, t.TBL_NAME name, t.TBL_TYPE, tp.PARAM_VALUE as description,
```

Comment:
I think `schema` would work .

I try out `select d.name as `schema` from DBS d` and could successfully connect to our hive metastore.

Reply:
Oh yeah, good point. I'll change this.